client/src/lib/dependency-injection/graphql/helpers.js

```diff
@@ -76,3 +86,112 @@ export const getFragments = ({ availableFragments, fragments = [] }) => (
         : capturedFragments
     ), '')
 );
+
+function getMatchingParamKey(matchWith, params) {
+  return Object.keys(params).find((paramName) => paramName.toLowerCase() === matchWith);
+}
+
+function getInputVariable({ templateName, singularName, params }) {
+  const inputKey = getMatchingParamKey('input', params);
+  if (inputKey) {
+    return `$${inputKey}: ${params[inputKey]}`;
+  }
```

Comment:
Allows for having graphql compatible variable declarations like `$input: CreateNoteInput!` by declaring the correct param name and type in the params object.